pypeit/core/extract.py

```diff
@@ -1501,6 +1514,9 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
         else:
             sobjs[iobj].FWHM = fwhm
 
+        # And boxcar_rad for skymask'ing.  This propogates to create_skymask_fwhm()
+        sobjs[iobj].boxcar_rad_pix = boxcar_rad_pix
+
```

Comment:
You need to add a similar line of code below to the all the hand apertures that are set later in this routine  i.e. starting around line 1580. Please go through the hand aperture logic there to make sure this parameter is set for all hand apertures.

Reply:
good catch. added